internal/cache/cache.go

```diff
@@ -0,0 +1,37 @@
+package cache
+
+import (
+	"context"
+	"eventapi/config"
+	"time"
+
+	"github.com/redis/go-redis/v9"
+)
+
+type CacheService struct {
+	client *redis.Client
+}
+
+func NewCacheService() *CacheService {
+	config := config.New()
+
+	return &CacheService{
+		client: redis.NewClient(&redis.Options{
+			Addr:     config.Cache.Address,
+			Password: "",
+			DB:       0,
+		}),
+	}
+}
+
+func (s *CacheService) Get(c context.Context, key string) (string, error) {
+	return s.client.Get(c, key).Result()
+}
+
+func (s *CacheService) Set(c context.Context, key string, value interface{}, ttl time.Duration) error {
```

Comment:
Just an FYI, `any` is an alias for `interface{}`

https://tip.golang.org/doc/go1.18#:~:text=The%20new%20predeclared%20identifier%20any%20is%20an%20alias%20for%20the%20empty%20interface.%20It%20may%20be%20used%20instead%20of%20interface%7B%7D.